tools/smina/smina.xml

```diff
@@ -0,0 +1,364 @@
+          --center_y $search_space.center_y
+          --center_z $search_space.center_z 
+          --size_x $search_space.size_x
+          --size_y $search_space.size_y
+          --size_z $search_space.size_z
+
+          #if '$search_space.autobox_ligand'
+              --autobox_ligand $search_space.autobox_ligand 
+          #end if 
+          #if '$search_space.autobox_add' 
+              --autobox_add $search_space.autobox_add
+          #end if
+          #if '$search_space.no_lig'=="true"
+              --no_lig
+          #end if 
+
+          --out ligand_out.pdbqt
+          #if '$output_sect.out_flex'=="true"
+              --out_flex output_flex
+          #end if                                                    
+          #if '$output_sect.log'=="true"
+              --log output_log
+          #end if
+          #if $output_sect.atom_terms=="true"
+              --atom_terms output_atom_terms
+          #end if
+          #if $output_sect.atom_term_data=="true"
+              --atom_term_data
+          #end if
+
+          --scoring $scoring_and_minimization.scoring 
+          #if $scoring_and_minimization.custom_scoring
+              --custom_scoring $scoring_and_minimization.custom_scoring
+          #end if
+          #if $scoring_and_minimization.custom_atoms
+              --custom_atoms $scoring_and_minimization.custom_atoms 
+          #end if
+          #if $scoring_and_minimization.score_only=="true"
+              --score_only 
+          #end if
+          #if $scoring_and_minimization.local_only=="true"
+              --local_only 
+          #end if
+          #if $scoring_and_minimization.minimize=="true"
+              --minimize 
+          #end if 
+          #if $scoring_and_minimization.randomize_only=="true"    
+              --randomize_only  
+          #end if 
+              --minimize_iters $scoring_and_minimization.minimize_iters 
+          #if $scoring_and_minimization.accurate_line=="true"
+              --accurate_line
+          #end if
+          #if $scoring_and_minimization.minimize_early_term=="true"
+              --minimize_early_term
+          #end if
+          #if $scoring_and_minimization.approximation
+              --approximation $scoring_and_minimization.approximation
+          #end if
+          #if $scoring_and_minimization.factor
+              --factor $scoring_and_minimization.factor
+          #end if
+          #if $scoring_and_minimization.force_cap
+              --force_cap $scoring_and_minimization.force_cap
+          #end if
+          #if $scoring_and_minimization.user_grid
+              --user_grid $scoring_and_minimization.user_grid
+          #end if
+              --user_grid_lambda $scoring_and_minimization.user_grid_lambda
+          #if $scoring_and_minimization.print_terms=="true"
+              --print_terms
+          #end if
+          #if $scoring_and_minimization.print_atom_types=="true"
+              --print_atom_types
+          #end if
```

Comment:
If you change the param to boolean below, you can replace the whole `#if` block with just:
```
          $scoring_and_minimization.print_atom_types
```